packages/shorebird_cli/test/src/commands/release_command_test.dart

```diff
@@ -253,8 +253,11 @@ Did you forget to run "shorebird init"?''',
     test('aborts when user opts out', () async {
       when(() => logger.confirm(any())).thenReturn(false);
       when(
-        () => logger.prompt(any(), defaultValue: any(named: 'defaultValue')),
-      ).thenReturn(appDisplayName);
```

Comment:
The only `logger.prompt` call in the release command is for the version number, so I think this may have been vestigial.